### ISSUE TITLE
ui5-dialog NVDA is reading the header text as clickable when it is not

### ISSUE THREAD

**Issue by bonniegso** [SAP/ui5-webcomponents#5618]
### **Bug Description**
When navigating in browse mode with the screen reader, NVDA, the header text is read as "clickable heading level 2" when the header text is not clickable.

### **Expected Behavior**
The screen reader should only read "heading NVDA should announce"heading level 2 <headerText>" when user navigates to the header text in screen reader browse modes.

### **Steps to Reproduce**
1. Go to https://stackblitz.com/edit/js-34awxd
2. Turn on NVDA screen reader
3. Navigate to the "Open Dialog" button using your keyboard and select it
4. Using your arrow keys navigate through the content on the dialog
5. Observe that "clickable heading level 2 This is a header" is read

### **Isolated Example**
https://stackblitz.com/edit/js-34awxd

### **Context**
- UI5 Web Components version: 1.5.0
- OS/Platform: Windows 10/NVDA
- Browser: Google Chrome
- Affected component: ui5-dialog

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SAP Successfactors
- Business impact: Accessibility

**Comment by MapTo0** [comment]
Dear Colleagues,

I have corrected the example: https://stackblitz.com/edit/js-w8ckdd?file=index.js,index.html

The issue is reproducible **only with NVDA, not with JAWS**.
I did not find any problem with the coding and this is probably a NVDA limitation.

Could you please with your ACC expert that the markup is correct and close this?

Regards,
Martin

**Comment by bonniegso** [comment]
Hi @MapTo0,

It is only reproducible on NVDA, however, I think this issue might be fixable. NVDA reads "clickable" if there is a click handler on the element. Can you please verify that there is no click hander for the dialog header?

Thanks,
Bonnie

**Comment by MapTo0** [comment]
@bonniegso 

In my opinion, it would be super strange if click handlers (javascript) affect the behaviour of a tool that reads the markup.
I believe this should not be the case here.

Regards!

**Comment by TeodorTaushanov** [comment]
Hi,

Please check the answer here:
https://github.com/SAP/ui5-webcomponents/issues/5601#issuecomment-1223939324

Best,
Teo
